### ISSUE TITLE
UI: Full page company description when no jobs listed

### ISSUE THREAD

**Issue by sindhus** [hasgeek/hasjob#255]
It would be nice to simply be shown company page if no jobs are listed for the company and a notification bar above saying "No jobs listed at the moment, check back later" or "Sign up to be notified"?

![hasjob_company](https://cloud.githubusercontent.com/assets/4291156/11330810/a60c3b80-91d7-11e5-93ef-5fcdf414e428.png)

**Comment by jace** [comment]
You're looking at search results, not the company's page. While we could redirect automatically, this will likely break instant refresh.

**Comment by jace** [comment]
@shreyas-satish In case search shows a single result, can we open that result (and change URL) without disrupting the filters, so the user can continue filtering?

**Comment by shreyas-satish** [comment]
Yeah, we could. But, is that a good idea from a UX perspective? I would be surprised if a search engine redirected me because there was exactly one result.

**Comment by sindhus** [comment]
Me things: In case it shows one or more results, then let this view be. If no results, redirect to company page.

**Comment by jace** [comment]
Company name matching is a result. If there are no results, there's nowhere to redirect to.

Unless you mean search matches a Domain but not a JobPost, and there is only one matching Domain.

**Comment by sindhus** [comment]
So there is no distinction between company pages and jobposts? All are internally a sticky/jobpost type object?

**Comment by sindhus** [comment]
> Unless you mean search matches a Domain but not a JobPost, and there is only one matching Domain.
Yep!

+ this view...

![fireshot capture 19 - mad street den hasjob - https___hasjob co_madstreetden com_archive 1](https://cloud.githubusercontent.com/assets/4291156/11394238/d9068082-9389-11e5-802d-875ed1fc722c.png)
